Task_HelloWorld/index.html

```diff
@@ -7,5 +7,11 @@
 
   <body style="margin: 0; padding: 0">
     <!-- Start coding below FOR HELLO WORLD TASK -->
+    <h>Hello World:</h>
+<p>In HTML, you can use various tags to format your content.
```

Comment:
The <br> tag should have a closing slash (/) before the closing angle bracket to be valid HTML.